modules/nw-sriov-dpdk-example-mellanox.adoc

```diff
@@ -105,7 +105,7 @@ spec:
     image: <DPDK_image> <2>
     securityContext:
```

Comment:
Please add "runAsUser: 0" under the securityContext

securityContext structure should look as follows: 
securityContext:
  runAsUser: 0
  capabilities:
    add: ["IPC_LOCK","SYS_RESOURCE","NET_RAW"]